service-02/PUS-2-12.asn1

```diff
@@ -0,0 +1,36 @@
+--!
+--! Copyright (C) 2017 N7 Mobile sp. z o. o.
+--! Contact: http://n7mobile.pl/Space
+--!
+--! This file is part of ASN.1/ACN PUS-C Components Library.
+--!
+--! Library was developed under a programme and funded by
+--! European Space Agency.
+--!
+--! This Library is free software: you can redistribute it and/or modify
+--! it under the terms of the GNU General Public License as published by
+--! the Free Software Foundation, either version 3 of the License, or
+--! (at your option) any later version.
+--!
+--! This Library is distributed in the hope that it will be useful,
+--! but WITHOUT ANY WARRANTY; without even the implied warranty of
+--! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--! GNU General Public License for more details.
+--!
+--! You should have received a copy of the GNU General Public License
+--! along with this program.  If not, see <http://www.gnu.org/licenses/>.
+--!
+
+PUS-2-12 DEFINITIONS AUTOMATIC TAGS ::= BEGIN
+EXPORTS ALL;
+IMPORTS
+    Transaction-ID, TransactionExecutionStatus FROM Transaction;
+
+TM-2-12-LogicalDeviceDataReport ::= SEQUENCE
+{
+    transaction-ID Transaction-ID,
+    transactionExecutionStatus TransactionExecutionStatus,
+    parameterValue OCTET STRING (SIZE(4)) --TODO
```

Comment:
why TODO? what should be here?

Reply:
It's dependent on transaction-ID, similarly  to TM-2-9 with transaction-ID and dataBlock

Reply:
Maybe TODO is not the best way to describe it. It's dependent on transaction-ID, so it probably cannot be modeled within ASN.1/ACN. It's a dummy type, similarly to dataBlock in PUS-2-9